homeassistant/components/bmw_connected_drive/lock.py

```diff
@@ -32,7 +32,13 @@ async def async_setup_entry(
 
     for vehicle in coordinator.account.vehicles:
         if not coordinator.read_only:
-            entities.append(BMWLock(coordinator, vehicle, "lock", "BMW lock"))
+            entities.append(
+                BMWLock(
+                    coordinator,
+                    vehicle,
+                    LockEntityDescription(key="lock", device_class="lock", name="Lock"),
```

Comment:
It looks like the description is constant. Then we should define it as a constant at the module level and just set it as the entity description in the init method below.

Reply:
There's no device class for lock entities. Please remove that.